unstructured_ingest/v2/processes/connectors/notion/connector.py

```diff
@@ -0,0 +1,342 @@
+import typing as t
```

Comment:
Can we use the explicit imports rather than an alias? i.e. `from typing import Optional`